### DISCUSSION TITLE
Welcome to xDrip Discussions!

### DISCUSSION THREAD

**Discussion by jamorham** [NightscoutFoundation/xDrip#1900] [Announcements]
## 👋 Welcome!
  Please see the directions below from @Navid200 on how to use the discussions feature and associated guidance and rules.

**Comment by Navid200** [comment]
Please use these discussions to discuss issues, ask questions, make suggestions, help those who need it, and share ideas in relation to xDrip.  

- If you have a question about how to use xDrip, please go ahead and ask.  
- If you have identified a bug, please post a description here.  Please don't open as issue until we ask you to.  
- If you have a feature request, please post a description here.  Please don't open  an issue until we ask you to.  
- If you have a suggestion for how we should run this repository, please post a discussion here.  
- Please engage in discussions and help those who need help if you can.  If you are experienced with xDrip and can help, your help will be appreciated.    
- If you are a programmer and like to contribute, please start a discussion and explain.  It's best to talk about your idea before spending too much time on it.  It is always possible a capability or a slightly different version of it may already exist.  
- If you are a programmer and need help getting started with xDrip, please discuss here.  
  
Click [here](https://github.com/NightscoutFoundation/xDrip/discussions/new/choose) to start one.  
  
</br>  
  
---  
  
#### **Please do not:**  
- Offer medical advice -  You are OK telling someone how to use xDrip.  But, if you tell someone how to control their blood glucose, you have gone too far.  You can say "estimated A1C is shown under statistics".  You cannot say "you should keep your A1C below 7".  Even if you are a Doctor of Medicine, you are not allowed to provide medical advice here.  
- Include links to xDrip forks.
- Include links to other sites unless absolutely necessary.  If you can make a point without including a link to a different site, please choose that.
- Spam  
- Bully  
- Offend - Any offensive remark including such based on sexual orientation, religion, ethnicity, and disability is strictly forbidden.  
- Discuss irrelevant matters to xDrip like politics, religion, sports, business opportunities, sales and or donations  
- Humor at the expense of someone else    
- Troll  
- Make personal attacks -  You can disagree as long as you focus on the subject.  But, please don't make it personal.  
- Discuss reverse engineering
- If you need help and prefer the conversation to be private, please reach out to us.  But, no one should reach out to you asking for your contact info.  If that happens, please don't provide it and please inform the mods (see below). 
  
</br>  
  
---  
  
#### **Attaching images**  
  
Before attaching an image, please edit it and scale it down.  Attaching a very large image will make it hard for anyone to go through the thread.  You can use a simple image editor for this.  For example, on Windows, you can use "Paint".  
Scale it down as much as you can such that if you scaled it down any more, it would not be visible or the  included text would not be legible.  
  
While you are editing your post, place the cursor where you want the image to appear.  Click on the paper clip symbol as shown in the image below.  
  
![Untitled](https://github.com/NightscoutFoundation/xDrip/assets/51497406/205d940d-9622-45a8-b8e4-3def73fce033)   
  
That will open up a form letting you choose the image file to attach.  
  
<br/>  
  
---  
  
#### **Posting through your email client**  
Please don't do that!
Even though it may seem much easier to just respond to the email you receive from GitHub.  Please don't.  If you do, the resulting post may include too much unnecessary content making it very hard for anyone to follow the conversation.  Please use a browser, log into GitHub, and respond to posts that way.  Thanks for your consideration.  
<br/>  
  
---
